src/components/Table/Table.tsx

```diff
@@ -119,43 +125,144 @@ const Tbody = styled.tbody`
     gap: 0.25rem;
   }
 `;
+
+const SelectData = styled.td`
+  overflow: hidden;
+  ${({ theme }) => `
+    color: ${theme.click.table.row.color.text.default};
+    font: ${theme.click.table.cell.text.default};
+    padding: ${theme.click.table.body.cell.space.md.y} ${theme.click.table.body.cell.space.md.x};
+  `}
+  @media (max-width: 768px) {
```

Comment:
don't think this breakpoint should be hardcoded.

Reply:
same as above